app/forms/mailing_address_form.rb

```diff
@@ -0,0 +1,7 @@
+class MailingAddressForm < QuestionsForm
```

Comment:
re: above comment about when to pull address from user to intake - we could also do that in this form in the `existing_attributes` class method